example/register_certificate/register_certificate.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	client_config "github.com/scalar-labs/scalardl-go-client-sdk/v3/client/config"
+	client_error "github.com/scalar-labs/scalardl-go-client-sdk/v3/client/error"
+	client_service "github.com/scalar-labs/scalardl-go-client-sdk/v3/client/service"
+)
+
+var propertiesFile = flag.String("properties", "client.properties", "the properties file")
+
+func main() {
+	flag.Parse()
+
+	var (
+		clientConfig client_config.ClientConfig
+		properties   []byte
+		err          error
+	)
+
+	if properties, err = ioutil.ReadFile(*propertiesFile); err != nil {
+		fmt.Println(err)
```

Comment:
Not that important for a sample but maybe just as a best practice, better to use the logger instead of `fmt.Println`.  I didn't know this as well but apparently fmt.println is not go-routine safe as well. (or it used to be)

Reply:
I didn't know that either.
Let me revise it with the log package.

Reply:
@ypeckstadt 

I use the log package to output the errors and also replace the use of `os.Exit` by just calling `log.Panic*`.

https://github.com/scalar-labs/scalardl-go-client-sdk/pull/20/commits/7d183b75dddd1d6a7b3f44e2ed025e27c92045a1

Please check again.
Thanks!

Reply:
Thank you.